ckan/controllers/user.py

```diff
@@ -485,12 +485,7 @@ def request_reset(self):
                         data_dict['id'] = user_list[0]['id']
                         user_dict = get_action('user_show')(context, data_dict)
                         user_obj = context['user_obj']
-                    elif len(user_list) > 1:
-                        h.flash_error(_('"%s" matched several users') % (id))
```

Comment:
Can you log this (`log.info()`) and the other messages so maintainers can still track it?